src/LoadingSparseTaskS3.cpp

```diff
@@ -45,13 +40,12 @@ void LoadingSparseTaskS3::check_loading(const Configuration& config,
                                         std::unique_ptr<S3Client>& s3_client) {
   std::cout << "[LOADER] Trying to get sample with id: " << 0 << std::endl;
 
-  std::string obj_id = std::to_string(hash_f(std::to_string(SAMPLE_BASE).c_str())) + "-CRITEO";
+  std::string obj_id =
+      std::to_string(hash_f(std::to_string(SAMPLE_BASE).c_str())) + "-CRITEO";
   std::string data =
       s3_client->s3_get_object_value(obj_id, config.get_s3_bucket());
 
   SparseDataset dataset(data.data(), true);
-  dataset.check();
```

Comment:
I think these checks are useful and should be kept unless there is a good reason to remove them